src/store/effects/keyboard.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get'
+import { nextChapterPlaytime, previousChapterPlaytime } from 'utils/chapters'
+
+import actions from '../actions'
+
+let modifier = 0
+
+const calcModifier = () => {
+  modifier = modifier < 7 ? modifier + 0.25 : 7
+  return modifier
+}
+
+const resetModifier = () => {
+  modifier = 0
+}
+
+const scrubForward = store => () => {
+  const state = store.getState()
+  const playtime = get(state, 'playtime')
+  const duration = get(state, 'duration')
+  let time = playtime + Math.pow(2, calcModifier())
+
+  time = time > duration ? duration : time
+
+  store.dispatch(actions.updatePlaytime(time))
+}
+
+const scrubBackward = store => () => {
+  const state = store.getState()
+  const playtime = get(state, 'playtime')
+  const duration = get(state, 'duration')
+  let time = playtime - Math.pow(2, calcModifier())
+
+  time = time > duration ? duration : time
```

Comment:
I think you need to check wether you scrub below the beginning of the track, here...